### PR TITLE
gh-125933: Add ARIA labels to select elements in the version switcher

### LINKED ISSUE #125933: Add ARIA labels to improve documentation accessibility
# Documentation

According to the Lighthouse report, select elements do not have labels.
image

These `select` elements are rendered in the JS in `Doc/tools/static/rtd_switcher.js`.

The issue was originally outlined on the [Discourse](https://discuss.python.org/t/docs-html-render-redesign/48566) (6th point).

### Linked PRs
* gh-125934
* gh-125938
* gh-125939

### PR DESCRIPTION
**Pull request by Privat33r-dev**
Fixes #125933

* Issue: gh-125933

----
📚 Documentation preview 📚: https://cpython-previews--125934.org.readthedocs.build/

### CODE REVIEW

File: Doc/tools/static/rtd_switcher.js [hunk 1]
```diff
@@ -7,7 +7,7 @@
  document.addEventListener("readthedocs-addons-data-ready", function(event) {
    const config = event.detail.data()
    const versionSelect = `
-   <select id="version_select">
+   <select id="version_select" aria-label="Python Version">
```
Comment (hugovk) [inline Doc/tools/static/rtd_switcher.js]:
```suggestion
   <select id="version_select" aria-label="Python version">
```

**Review by hugovk — APPROVED** [review]
Thank you!

**Comment by Privat33r-dev** [comment]
Thank you for quick review! 👍🏻

**Comment by hugovk** [comment]
This PR only applies to the docs previews on Read the Docs, we'll need a similar one in the https://github.com/python/docsbuild-scripts repo to take effect on https://docs.python.org.

Please could you open a PR over there too?

**Comment by Privat33r-dev** [comment]
> This PR only applies to the docs previews on Read the Docs, we'll need a similar one in the https://github.com/python/docsbuild-scripts repo to take effect on https://docs.python.org.
> 
> Please could you open a PR over there too?

Sure, https://github.com/python/docsbuild-scripts/pull/223

Thank you for explanation and guidance. Also it makes me wonder: why don't buildscripts take templates from the same place as the Python's docs?

**Comment by hugovk** [comment]
Legacy reasons :) docsbuild-scripts came first, and they're both getting the data about what to put into the language/version switchers from different places.

There's not much overlap between these two files, and the RTD stuff is still work-in-progress, so it's probably better to let it settle first.

**Pull request merged**